src/Tasks.js

```diff
@@ -0,0 +1,33 @@
+export default class Tasks {
+  static index = 0;
+
+  constructor() {
+    this.tasks = JSON.parse(localStorage.getItem('tasks')) || [];
+  }
+
+  addTask = (description, completed) => {
+    this.tasks.push({
+      description,
+      completed,
+      index: parseInt(Tasks.index, 10),
+    });
+  };
+
+  deleteTask = (index) => {
+    this.tasks.forEach((val, i) => {
+      if (this.tasks[i].index === index) {
+        this.tasks.splice(i, 1);
+      }
+      Tasks.index -= 1;
+    });
+  };
+
+  editTask = (index, description) => {
+    this.tasks.forEach((val, i) => {
+      if (this.tasks[i].index === index) {
+        const temp = { ...this.tasks[i], description };
+        this.tasks[i] = temp;
+      }
+    });
+  };
```

Comment:
- [ ] Edit function should edit only the selected task but see what is happening, here is a list of tasks
![edit](https://user-images.githubusercontent.com/44745855/165821832-742beb58-400b-4da3-b2ae-4828b1213688.png)

see what happens when I edit one task it is actually changing all tasks , please address the issue
![al](https://user-images.githubusercontent.com/44745855/165822010-1d0e13e0-2103-4a58-bc74-56ee4cb1d979.png)